rules/builds.smk

```diff
@@ -693,6 +693,16 @@ def export_title(wildcards):
     # TODO: maybe we could replace this with a config entry for full/human-readable build name?
     location_name = wildcards.build_name
 
+    title = None
```

Comment:
If we `return` the title values from the config here instead of assigning them to a variable, we can delete the lines assigning title to `None` and checking for title downstream.